plugins/cairo-lang-quote/src/lib.rs

```diff
@@ -1,12 +1,12 @@
 use std::iter::Peekable;
 
-use proc_macro::{Delimiter, TokenStream as RustTokenStream, TokenTree as RustTokenTree};
-use proc_macro2::{Ident, Span};
+use proc_macro2::{Delimiter, Ident, Span, TokenTree};
```

Comment:
We use `quote!` anyway, which works on `proc_macro2`, using it throughout the code allows us for easier testing, as `proc_macro2` can be used wherever, while `proc_macro` only in a procedural macro code.